Client/mods/deathmatch/logic/CClientModelManager.cpp

```diff
@@ -60,7 +60,7 @@ int CClientModelManager::GetFirstFreeModelID(void)
     for (int i = 0; i < MAX_MODEL_ID; i++)
     {
         CModelInfo* pModelInfo = g_pGame->GetModelInfo(i, true);
-        if (!pModelInfo->IsValid())
+        if (!pModelInfo->IsValid() && (i < 300 || i > 319) )
```

Comment:
if this is going to be merged pls can we include a code comment that explains what this is protecting against? someone who reads this in the future might get confused as to what this is guarding

Reply:
Added